educator/tests/Test/Dscp/Educator/Web/Student/Queries.hs

```diff
@@ -91,7 +95,7 @@ spec_Student_API_queries :: Spec
 spec_Student_API_queries = specWithTempPostgresServer $ do
   describe "Courses" $ do
     describe "getCourse" $ do
-        it "Student is not enrolled initially" $
+        xit "Student is not enrolled initially" $
```

Comment:
Apparently, in order for tests to pass, you need to tell Tasty explicitely to behave well in case of `pending` tests.
This can be done by putting `{-# OPTIONS_GHC -F -pgmF tasty-discover -optF --tree-display -optF --treat-pending-as=success #-}` in `educator/tests/Main.hs`